docs/tutorial.rst

```diff
@@ -1,16 +1,67 @@
 Start Here: ``datascience`` Tutorial
-======
+====================================
 
-In progress.
+This is general introduction to the functionality in
+:py:mod:`datascience`.  For a reference guide, please see
+:ref:`tables-overview`.
 
 Introduction
-------
+------------
+
+First, load the :py:mod:`datascience` module:
+
+.. ipython:: python
+
+   import datascience as ds
+
+In the IPython notebook, type `ds.` followed by the TAB-key to see a list of members.
+The most important of these is the :py:class:`Table` class, which is
+the structure used to represent columns of data.
```

Comment:
This is considered poor practice in the Python community, so I'd strongly
recommend this syntax (same is being used by numpy, pandas, scipy,
matplotlib, etc.)

I understand about this semester potentially being problematic, though!